### ISSUE TITLE
RSS for individual prods/bbs threads

### ISSUE THREAD

**Issue by matiasw** [pouetnet/pouet2.0#58]
Hi,
I'd like to see a feature where I can subscribe to new comments on a prod, or to new posts on a BBS topic by RSS/Atom. I see you recently did some work on the RSS functionality; I'd be willing to do the work for implementing this feature, but I'm afraid with my lack of web technology skills, I'd need some amount on hand-holding (IRC hotline..?). That said, I _have_ created an RSS generator, once... 
Do you think this is a feature that could be added to pouët? Would you be available for help in its development?

(this ought to be labeled as enhancement; not sure why I can't apply the label, myself - maybe because I'm not a project contributor?)

**Comment by matiasw** [comment]
Actually, it seems that this functionality is available using a 3rd party service, without any modifications to pouët: http://page2rss.com/

**Comment by Gargaj** [comment]
Ehe, I'd much rather not have a web-scraper service hit Pouet.

**Comment by matiasw** [comment]
I looked it up and different sources said different things, but page2rss apparently hits monitored pages once every 2 hours - once a day. That's not  a huge load, is it?
The scraper works for me. Therefore, I'm withdrawing my offer of trying to implement this as a pouet feature. Sorry, but having to (re)learn stuff I'm not particularly interested in (Web) just doesn't seem as appealing when I've got a dirty hack that gets it done.
